website/views.py

```diff
@@ -186,9 +186,9 @@ def find_bookmark_collection(user):
 def dashboard(auth):
     return redirect('/')
 
-@ember_flag_is_active('ember_support_page')
+
 def support():
-    return {}
+    return use_ember_app()
```

Comment:
Caveat, you said you were having some trouble with this so could indeed be this is the way to do it!

It seems to me how this should work is that the route `/support/` is deleted entirely from v1 routes, this v1 `support()` view is deleted entirely, and the route is instead added to the list in `website/ember_osf_web/views.py`, maybe with some additional config to make that work right, not sure.

Nowhere else outside of the `ember_flag_is_active` decorator do we simply return `use_ember_app()`, and keeping around the v1 route and view seems like doing this halfway.

Reply:
😄🌴, yeah that works